cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/tools-control.tsx

```diff
@@ -1022,6 +1022,45 @@ export class ToolsControlComponent extends React.PureComponent<Props, State> {
             });
         });
 
+        function checkAttributesCompatibility(
+            functionAttribute: ModelAttribute | undefined, // model
+            dbAttribute: any, // job
+            value: string,
+        ): boolean {
+            if (functionAttribute === undefined || dbAttribute === undefined) {
+                return false;
+            }
+
+            if (functionAttribute.input_type === 'checkbox') {
+                return ['true', 'false'].includes(value);
+            }
+
+            if (functionAttribute.input_type === dbAttribute.inputType) {
+                if (functionAttribute.input_type === 'number') {
+                    return !Number.isNaN(value);
+                }
+
+                return ['select', 'radio', 'text'].includes(functionAttribute.input_type);
+            }
+
+            switch (functionAttribute.input_type) {
+                case 'number':
+                    return ['select', 'radio', 'text'].includes(dbAttribute.input_type) && !Number.isNaN(value);
```

Comment:
I believe `dbAttribute` is an instance of `cvat-core` `Attribute` class, so shouln't we check for `dbAttribute.inputType` instead?

Reply:
The object is typed as `any` in the code, so I examined the object structure by debugging the code. The attribute name is snake cased.